src/lib/constants.ts

```diff
@@ -4,4 +4,6 @@ export const APP_DESCRIPTION =
 
 export const STORAGE_KEY = 'OPENAI_API_KEY';
 
+export const STORAGE_URL = 'http://ai-api.e-tudou.com:9000/v1';
```

Comment:
@klb3713 This const is used for browser storage, would like to change this key and value to 
`OPENAI_API_BASE_URL`.

Reply:
yes, sorry, I forgot to modify this default, it's our internal service